gpii/node_modules/flowManager/src/UserLogonStateChange.js

```diff
@@ -159,8 +159,7 @@ fluid.defaults("gpii.flowManager.userLogonHandling.stateChangeHandler", {
         },
         errorResponse: {
             funcName: "gpii.flowManager.userLogonHandling.errorResponse",
-            args: ["{request}", "{arguments}.0", "{arguments}.1"]
-                              // message,         statusCode
+            args: ["{request}", "{arguments}.0", "{arguments}.1"] // message, statusCode
```

Comment:
See below - this is an unacceptable change

Reply:
Fixed.